doc/release-notes/1.0/1.0.10.rst

```diff
@@ -0,0 +1,43 @@
+======================
+ Percona Agent 1.0.10
+======================
+
+Percona is glad to announce the release of Percona Agent 1.0.10 on January 6th 2015. Downloads are available from our download site `here <http://www.percona.com/downloads/percona-agent/1.0.10>`_. Percona Agent can be `installed <http://cloud-docs.percona.com/Install.html>`_ or `upgraded <http://cloud-docs.percona.com/Install.html#updating-the-agent>`_ as described in the `manual <http://cloud-docs.percona.com/index.html>`_. 
+
+New Features
+------------
+
+ * *Percona Agent* has improved the data sender timeout handling.
+
+ * New ``--status`` option has been implemented which can be used to get information on agent's status via API
+
+ * *Percona Agent* now enables the data compression when it gets installed.
+
+ * New ``--agent-mysql-user`` and ``--agent-mysql-pass`` options have been implemented to support installing agent with already existing user which has proper privileges on the server.
+
+ * *Percona Agent* now provides user-friendly error messages with additional details for troubleshooting, instead of general ``Error has occured`` message. 
```

Comment:
This is a very broad statement, but the change is much more specific: better error messages just when connections to MySQL fail (ask @cfsalguero for more details).